(tlg4102)tlg4102.tlg001_tlg4102.tlg007

AlisonBabeu opened the pull request:
Updated cts descriptions, fixed dates in metadata and added in page level links.

lcerrato commented on data/tlg4102/tlg001/__cts__.xml:
```diff
@@ -1,8 +1,8 @@
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr. 23), 
-    Catenæ Græcorum Patrum in Novum Testamentum, Vol 1, Cramer, Oxford 1840</ti:description>
+    <ti:label xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr.)</ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 1.
```
We don't generally use the dipthong (we would just transcribe to ae), but if you do, then I think you may need to have a Unicode declaration in the file.

lcerrato commented on data/tlg4102/tlg001/__cts__.xml:
```diff
@@ -1,8 +1,8 @@
-    Catenæ Græcorum Patrum in Novum Testamentum, Vol 1, Cramer, Oxford 1840</ti:description>
+    <ti:label xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr.)</ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 1.
+      Cramer, John Anthony, editor. Oxford: Oxford Unviersity Pres, 1840.</ti:description>
```
University misspelled

AlisonBabeu replied:
Doh. Thanks. I will fix that as well as the changing the dipthong above!

lcerrato commented on data/tlg4102/tlg003/__cts__.xml:
```diff
@@ -1,8 +1,8 @@
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium 
-      sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4), Cramer, Oxford, 1844 </ti:description>
+    <ti:label xml:lang="lat">Supplementum et varietas lectionis ad commentarium Chrysostomi in evangelium sancti Matthaei (e cod. Oxon. Bodl. Auct. T.1.4)</ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 1.
```
Same two issues as above.

lcerrato commented on data/tlg4102/tlg004/__cts__.xml:
```diff
@@ -2,7 +2,7 @@
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Lucam (Typus B) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Misc. 182),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
+      Cramer, John Anthony, editor. Oxford: Oxford Unviersity Pres, 1841.</ti:description>        
```
same as above (æ versus ae) and "Unviersity"

lcerrato commented on data/tlg4102/tlg007/__cts__.xml:
```diff
@@ -1,8 +1,9 @@
-    Cramer, Oxford, 1844</ti:description>
+    <ti:label xml:lang="lat">Supplementum Et Varietas Lectionis Ad Catenam In Evangelium Sancti Joannis (e cod. Bodl. B.)
+    </ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
```
same as other CTS files

lcerrato commented on data/tlg4102/tlg006/__cts__.xml:
```diff
@@ -2,7 +2,7 @@
     <ti:label xml:lang="lat">Supplementum et varietas lectionis ad catenam in evangelium sancti Lucae (e cod. Oxon. Bodl. Laud. 33)</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Supplementum et varietas lectionis ad catenam in evangelium sancti Lucae (e cod. Oxon. Bodl. Laud. 33),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
```
same as other cts files

lcerrato commented on data/tlg4102/tlg005/__cts__.xml:
```diff
@@ -2,7 +2,7 @@
     <ti:label xml:lang="lat">Catena In Joannem (Catena Integra) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Auct. T.1.4)</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Joannem (Catena Integra) (e codd. Paris. Coislin. 23 + Oxon. Bodl. Auct. T.1.4),
-    Cramer, Oxford, 1844</ti:description>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 2.
```
same as other CTS files

lcerrato commented:
Hi @AlisonBabeu 
In all of these CTS files there are two minor issues:

1. "University" is misspelled
2. the diphthong is used instead of ae.  This is an editorial choice (I usually transcribe to two letters for simplicity) but it may be that if the dipthong is maintained that we want to be sure that a Unicode declaration is in every CTS file. This is inconsistent across all collections: ` <?xml version="1.0" encoding="UTF-8"?>`

So the typo needs changing but the dipthong can stay based on how you want this to appear. We should do a check on the Unicode declarations, though.

AlisonBabeu commented:
hey @lcerrato so I've changed the dipthongs to ae and fixed the spelling, and will not use the dipthong moving forward. Looking at the cts_.xml files for this textgroup question I don't see any Unicode declaration. In fact none of the cts_.xml files have the XML top level root element either or I am only supposed to find that in the TEI-XML files? Thanks!

lcerrato commented:
@AlisonBabeu 
We have some in Perseus but I do not know if there was ever a standard practice established on that. Without it, it's very likely the special characters would break.
We could probably test this in the cts reading environment, but it's not an urgent issue.
